packages/protocol/test/governance/lockedgold.ts

```diff
@@ -412,4 +413,37 @@ contract('LockedGold', (accounts: string[]) => {
       })
     })
   })
+
+  describe('#slash', () => {
+    const value = 1000
+    const reporter = accounts[2]
+
+    beforeEach(async () => {
+      // @ts-ignore: TODO(mcortesi) fix typings for TransactionDetails
+      await lockedGold.lock({ value })
+      await lockedGold.addSlasher(account)
+    })
+
+    describe('when the account is slashed for all of its locked gold', () => {
```

Comment:
Add a test for when some of the gold that gets slashed is voting

Reply:
In the middle of this now, but my reasoning for not including this originally was because this seems like a better fit for e2e testing.
The slowdown right now is figuring out how to get the necessary functions out of the `election` contract within the `lockedgold` tests.